nextcloud-app-notify_push/riscv64.patch

```diff
@@ -0,0 +1,28 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -18,9 +18,11 @@ install="$pkgname.install"
+ groups=(nextcloud-apps)
+ _archive="${pkgname##*-}-$pkgver"
+ source=("$url/archive/v$pkgver/$_archive.tar.gz"
+-        "$pkgname.service")
++        "$pkgname.service"
++        "$pkgname-add-sleep.patch::https://github.com/SpriteOvO/notify_push/commit/aa2704273cc07a95dcb7d1e671b5115f71e01188.patch")
```

Comment:
I'd suggest changing this to an upstream url, *i.e.* https://github.com/nextcloud/notify_push/pull/171.patch

Reply:
Sorry, that was a mistake. Fixed now.